science-dev-board/Core/Src/main.c

```diff
@@ -52,6 +65,19 @@ UART_HandleTypeDef huart1;
 
 /* USER CODE BEGIN PV */
 
+ADCSensor* adc_sensor = NULL;
+Bridge* bridge = NULL;
+Current_Sensor* current_sensor = NULL;
+Temp_Sensor* temp_sensor = NULL;
+Heater* heater = NULL;
+PinData* debug_led = NULL;
+PinData* MOSFETDevices = NULL;

```

Comment:
Should be an array of certain sizes.. also use underscore and just make Moffett lowervase